recipes/nanoplot/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.42.0" %}
+{% set version = "1.43.0" %}
 
 package:
   name: nanoplot
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/n/nanoplot/NanoPlot-{{ version }}.tar.gz
-  sha256: 0f8fd2cffd33a346b3306716058c6cb4091c931e8ab502f10b17a28749e8b6d9
+  url: https://files.pythonhosted.org/packages/5f/95/8db2c1dfa964fdf85f708e8986a0f49ed6e8f0787c4a24ad01a84dded3c8/nanoplot-{{ version }}.tar.gz
```

Comment:
Unfortunately the older version based URL does not work.

Reply:
https://pypi.io/packages/source/n/nanoplot/nanoplot-1.43.0.tar.gz works!

Reply:
Damn! Tricked by the capitalization. I will correct this.